clang/include/clang/Sema/Sema.h

```diff
@@ -14040,6 +14040,8 @@ class Sema final {
   bool SemaValueIsRunOfOnes(CallExpr *TheCall, unsigned ArgNum);
 
 public:
+  bool SemaIsLayoutCompatible(QualType T1, QualType T2);
```

Comment:
Can this function no be `const`? And I guess the "Sema" prefix is to disambiguate with the existing `isLayoutCompatible`?

Reply:
Agreed, this should be renamed IsLayoutCompatible (and be const)

Reply:
Done